arches/app/media/js/viewmodels/reference-select.js

```diff
@@ -45,17 +30,37 @@ define([
             return val ? name : null;
         });
 
-        this.selectionValue.subscribe(val => {
-            if (val) {
-                if (!(val instanceof Array)) { val = [val]; }
-                const tileReady = val.map(uri => {
-                    return {
-                        "uri": uri,
-                        "listid": NAME_LOOKUP[uri]["listid"],
-                        "labels": NAME_LOOKUP[uri].labels
-                    };
-                });
-                self.value(tileReady);
+        this.valueAndSelectionDiffer = function(value, selection) {
+            if (!(ko.unwrap(value) instanceof Array)) {
```

Comment:
I found that `selection` also needed to be unwrapped in the case of re-selecting the same value.

Reply:
I still need a little help with this one :pray:

`this.valueAndSelectionDiffer` is called twice:
- one caller (selectionValue subscription) provides `selection` a list of strings 
- the other (value subscription) provides `selection` an observable

JSON.stringify'ing an observable will always be undefined, so why is this the value subscription calling this if it's always going to return false?